src/main/java/by/itechart/api/entity/UserRole.java

```diff
@@ -15,6 +18,7 @@
     private Role role;
 
     @OneToOne(mappedBy = "userRole")
+    @JsonManagedReference
```

Comment:
Since we introduced DTO classes do we still need this annotation here?

Reply:
> Since we introduced DTO classes do we still need this annotation here?

You are right! We don't need it anymore :)